modules/ROOT/pages/download/index.adoc

```diff
@@ -37,11 +37,11 @@ for important requirements for download pages for Apache projects.
 
 Apache NetBeans is released four times a year. For details, see link:https://cwiki.apache.org/confluence/display/NETBEANS/Release+Schedule[full release schedule].
 
-== Apache NetBeans 23
+== Apache NetBeans 24
 
-Latest version of the IDE, released on September 19, 2024.
+Latest version of the IDE, released on November 25, 2024.
```

Comment:
todo

Reply:
december 10, we keep the year that's cool

Reply:
note for future me: there is a second date string on the DL page